File: test_headers.js
const fetch = require('node-fetch');

async function testHeaders() {
    const res = await fetch("http://127.0.0.1:8000/login");
    console.log("Status:", res.status);
    console.log("Headers:");
    for (const [key, value] of res.headers) {
        console.log(`  ${key}: ${value}`);
    }
}

testHeaders();

File: test_headers.py
import urllib.request
import sys

try:
    response = urllib.request.urlopen('http://127.0.0.1:8000/login')
    print("STATUS:", response.getcode())
    print("HEADERS:")
    for key, value in response.info().items():
        print(f"{key}: {value}")
except Exception as e:
    print(f"Error: {e}")
